app/views/layouts/application.html.haml

```diff
@@ -8,6 +8,7 @@
       %meta{content: "noindex", :name => "robots"}/
     %meta{content: "#{t("layout.meta_description")}", name: "description"}/
     %meta{content: "IE-edge,chrome=1", "http-equiv" => "X-UA-Compatible"}/
+    %link{href: "https://www.nzsl.nz", :rel => "canonical"}/
```

Comment:
Like @joshmcarthur mentioned, we've got a base_url config variable setup for the href part of this now. It can be changed to Rails.application.config.base_url instead of "https://www.nzsl.nz"

Reply:
Ok thanks